classes/class-responsive-block-editor-addons-frontend-styles.php

```diff
@@ -14501,7 +14501,7 @@ public static function get_responsive_block_popup_css( $attr, $id ) {
 		}
 
 		/**
-		 * Get Defaults for Contact Form 7 Styler block
+		 * Get Defaults for Popup block.
```

Comment:
should be form block?

Reply:
While making Popup Block, I forgot to change the description, So, I did it now.
The function is of popup block, not of form block.